### ISSUE TITLE
Before create event event to reject or accept creating events

### ISSUE THREAD

**Issue by AhmedNourJamalElDin** [antoniandre/vue-cal-v4#370]
Hi,

Is it possible to reject an event after the user has selected the time slot he needs?
i.e. I want to validate the event and see if there is an overlapping with a maximum of 2 events, if so, I want to reject creating a new event.

To summarize, do we have such events:
- before dragging or creating to reject the creation e.g. I want to allow a maximum of 1 event at a time i.e. the user can create one and only one event at a time.
- before creating but after dragging to reject or accept an event i.e. if an event meets some condition, I want to accept it, otherwise, I want to reject it.
- on create an event successfully so I can know which event is created (only new events, not all events)

**Comment by AhmedNourJamalElDin** [comment]
Hi again,
I also need to reject some resizing event. i.e. if resizing and meet some condition, I want to revert the event to its original state.

I tried to implement it but I didn't know what to do exactly.

**Comment by Csandori** [comment]
Hi!

Try the onEventCreate prop. If you return false, then the creation is cancelled. 
For the resizing: event-duration-change event
For the dragging: event-drop

You can find all the informations in the documentation.

**Comment by AhmedNourJamalElDin** [comment]
Thanks for your answer.

> Try the onEventCreate prop. If you return false, then the creation is cancelled.

I meant at the time of clicking. e.g. if a user clicks at a specific time, I don't want to accept his click and show him some feedback that he can't select this time.
this must happen before dragging.

> For the resizing: event-duration-change event

Can I reject a resizing and revert the event back to its original state?

I'm sorry but I couldn't find such things in the documentation.

**Comment by Csandori** [comment]
You can find everything here:
https://jsfiddle.net/qm5acdw7/

I hope it helps :)

**Comment by vesper8** [comment]
@AhmedNourJamalElDin I want to do the exact same thing, did you figure it out? Would you mind sharing your solution if you did please? Many thanks!

**Comment by antoniandre** [comment]
Duplicate of #398, please continue thread in there.
